@here/harp-omv-datasource/lib/OmvData.ts

```diff
@@ -534,32 +497,26 @@ export class OmvProtobufDataAdapter implements OmvDataAdapter, OmvVisitor {
         }
 
         const geometry: ILineGeometry[] = [];
-        let positions: Vector3[];
-        this.m_geometryCommands.accept(
-            feature.geometry,
-            this.m_tileKey,
-            this.m_geoBox,
-            this.m_layer.extent,
-            {
-                type: "Line",
-                visitCommand: command => {
-                    if (isMoveToCommand(command)) {
-                        positions = [command.position];
-                        geometry.push({ positions });
-                    } else if (isLineToCommand(command)) {
-                        positions.push(command.position);
-                    }
+        let positions: Vector2[];
+        this.m_geometryCommands.accept(feature.geometry, {
```

Comment:
Is this code properly formatted?

Reply:
Yeah, if not it would have failed the ``yarn prettier`` step in CI.